__tests__/app.test.js

```diff
@@ -144,5 +144,49 @@ describe("All GET /api Tests", () => {
       });
   });
 
+  //POST TEST
+  //task 7
+  describe("POST /api/articles/:article_id/comments", () => {
+    test("status: 201 - should respond with the posted comment", () => {
+        const testComment = {
+            username: "butter_bridge",
+            body: "a new comment",
+        }
+        return request(app)
+            .post("/api/articles/11/comments")
+            .send(testComment)
+            .expect(201)
+            .then(({ body }) => {
+                const { comment } = body
 
+                expect(typeof comment).toBe("object")
+                expect(Object.keys(comment)).toHaveLength(6)
```

Comment:
This is a nice idea, but makes your test more brittle. This is because if you ever wanted to add a new column to this table it would then fail this test because of this specific assertion. We want to try and make tests as flexible as possible, and this is a nice idea but the below expect is the more robust and flexible way of checking the properties